src/vorta/scheduler.py

```diff
@@ -96,21 +114,139 @@ def next_job_for_profile(self, profile_id):
         else:
             return job.next_run_time.strftime('%Y-%m-%d %H:%M')
 
-    def create_backup(self, profile_id):
+    def enqueue_create_backup(self, profile_id, repo_id, vorta_queue):
+        vorta_queue.add_job(CreateJobSched(profile_id=profile_id, site=repo_id))
+
+
+class JobStatus(Enum):
```

Comment:
I wonder if all your job scheduling classes would better fit in the `borg` package, since they deal with running Borg commands? The `scheduler.py` module is just one more way to add jobs, but not much different from pressing the *Start* button. Maybe add your classes in `borg/jobs_manager.py`?

Reply:
I can because there are too much code in scheduler.py and no link between implementation. 
But maybe I can let the jobsManager initialization in the scheduler. In some way, the jobsManager schedule jobs too.

Reply:
Yes, that would be great! I know the name and concept of `scheduler.py` is similar, but what you did for the execution of Borg threads is more fundamental.

I also plan on making some changes to the actual Scheduler later. So the code there needs to focus on scheduling the next job, scheduling missed jobs, etc: https://github.com/borgbase/vorta/pull/908

Reply:
Done